Sources/Stevenson/SlowClient.swift

```diff
@@ -0,0 +1,160 @@
+/// Service to create a Slow / Rate-Limiting-Aware Client, especially useful for 3rd-party APIs that have quotas or can't handle too many requests in parallel
```

Comment:
I wish we were using ReactiveSwift.

Reply:
Yeah I searched inside the Vapor API if there were something built-in to throttle but doesn't seem like it 😢 
Went to the Vapor discord to ask for solutions and that `SlowClient` gist was the main solution – suggested by an user who seems quite active in the Vapor community and seemed to know what they were doing, so 🤷‍♂